From c650f6b0c75233bb2a669704829303cab301a8f8 Mon Sep 17 00:00:00 2001
From: Christian Jann <christian.jann@ymail.com>
Date: Sat, 14 Dec 2013 16:35:25 +0100
Subject: [PATCH] ARM: bones-badb: Removed duplicated ssp1

Removed duplicated ssp1 section in device tree, now the second SD card
works.
---
 arch/arm/boot/dts/imx23-olinuxino.dts | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/arch/arm/boot/dts/imx23-olinuxino.dts b/arch/arm/boot/dts/imx23-olinuxino.dts
index 759356a..90f8d43 100644
--- a/arch/arm/boot/dts/imx23-olinuxino.dts
+++ b/arch/arm/boot/dts/imx23-olinuxino.dts
@@ -65,15 +65,6 @@
 					fsl,pull-up = <0>;
 				};
 			};
-
-			ssp1: ssp@80034000 {
-				#address-cells = <1>;
-				#size-cells = <0>;
-				compatible = "fsl,imx23-spi";
-				pinctrl-names = "default";
-				pinctrl-0 = <&spi2_pins_a>;
-				status = "okay";
-			};
 		};
 
 		apbx@80040000 {
-- 
1.8.3.1

